pkg/operator/ceph/csi/controller.go

```diff
@@ -299,3 +303,21 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 
 	return reconcileResult, nil
 }
+
+func setCSILogrotateParams(cephClustersItems []cephv1.CephCluster) {
+	logger.Debugf("set logrotate values in csi param")
+	spec := cephClustersItems[0].Spec
```

Comment:
check the length before accessing the index `0`

Reply:
We already checked for the length of it in the re-concile, and if there is no cephcluster we return the csi rconcile